build_tools/ci/cpu_comparison/run.py

```diff
@@ -537,8 +537,8 @@ def aie_vs_llvm_cpu(
     config,
     test_file,
     use_ukernel=False,
-    tile_pipeline="pack-peel",
-    lower_to_aie_pipeline="objectFifo",
+    tile_pipeline="pad-pack",
+    lower_to_aie_pipeline="air",
```

Comment:
Can you leave the default as "pack-peel" and "objectFifo"? You can add a test for pad-pack pipeline if you like, but I think the current tests are enough.